example/lib/pages/tab_view.dart

```diff
@@ -8,8 +11,65 @@ class ZenitTabViewExample extends StatefulWidget {
 }
 
 class _ZenitTabViewExampleState extends State<ZenitTabViewExample> {
+  late int _selectedIndex;
```

Comment:
you could just do `int _selectedIndex = 0;` so you can remove late and initState